projects/organization-management/src/app/store/users/users.reducer.ts

```diff
@@ -45,8 +48,16 @@ const initialState: UsersState = usersAdapter.getInitialState({
 
 export const usersReducer = createReducer(
   initialState,
-  setLoadingOn(loadUsers, addUser, updateUser, deleteUser),
-  setErrorOn(loadUsersFail, loadUserFail, addUserFail, updateUserFail, deleteUserFail, setUserRolesFail),
+  setLoadingOn(loadUsers, addUser, updateUser, deleteUser, setUserBudgets),
```

Comment:
I don't think loading should be set explicitely for the budget update. For creation it is concatenated in the service and for editing we do optimistic updates.